### DISCUSSION TITLE
Multi-scale training and Multi-scale test

### DISCUSSION THREAD

**Discussion by luooofan** [rwightman/efficientdet-pytorch#215] [General]
How to use multi-scale training and multi-scale test?
Are these technologies implemented already?

**Comment by rwightman** [comment]
@Luoofan multi-scale training as described in the paper and used in the TF impl is included here by default. I did not include any multi-scale test but I'm pretty sure others have used the models with multi-scale test.

The next augmentations I would like to add are mosaic and some sort of obj detection focused Auto/RandAugment. Not sure when I will find the time to get to that though...
